mlir-sycl/lib/Conversion/SYCLToLLVM/DPCPP.cpp

```diff
@@ -1978,10 +1978,12 @@ class NDItemGetGroupLinearIDPattern
         cast<NdItemType>(op.getNDItem().getType().getElementType())
             .getBody()[indices[0]];
     const auto convGroupTy = getTypeConverter()->convertType(groupTy);
+    Type ndItemTy = op.getNDItem().getType().getElementType();
```

Comment:
[nit] Can you please change line 1977 and 1981 to:
```
NdItemType ndItemTy = cast<NdItemType>(op.getNDItem().getType().getElementType());
const auto groupTy = ndItemTy.getBody()[indices[0]];
```

Reply:
Done